apps/web/src/components/care-terminologies/common-search-terms.tsx

```diff
@@ -0,0 +1,33 @@
+import { debounce } from 'lodash';
```

Comment:
It looks like a bunch of imports are not being used in this function.

<img width="687" alt="Screenshot 2023-12-15 at 1 17 07 PM" src="https://github.com/bcgov/MOH_TeamBasedCare/assets/54554766/9e587d1d-4ed2-41f5-9c77-96bffc2895a4">